xml/shared/src/main/scala/scala/xml/persistent/CachedFileStorage.scala

```diff
@@ -96,7 +97,7 @@ abstract class CachedFileStorage(private val file1: File) extends Thread {
     val c = fos.getChannel()
 
     // @todo: optimize
-    val storageNode = <nodes>{ nodes.toList }</nodes>
+    val storageNode = <nodes>{nodes.toList}</nodes>
```

Comment:
This file, CachedFileStorage.scala, also only has whitespace changes that should be preserved for now, IMO